src/netCDF4/_netCDF4.pyx

```diff
@@ -2653,8 +2653,9 @@ length greater than one are aliases for `str`.
 Data from netCDF variables is presented to python as numpy arrays with
 the corresponding data type.
 
-`dimensions` must be a tuple containing dimension names (strings) that
-have been defined previously using `Dataset.createDimension`. The default value
+`dimensions` must be a tuple containing `Dimension` instances or
```

Comment:
perhaps to clarify that a mix is allowed, `a tuple containing mixed `Dimension`. Perhaps it's just me ;)

Reply:
use and/or instead of or